lineage/tHMM.py

```diff
@@ -97,8 +97,12 @@ def get_AIC(self, LL):
         if self.fT is None:
             degrees_of_freedom += self.num_states * (self.num_states - 1)
 
-        for ii in range(self.num_states):
-            degrees_of_freedom += self.estimate.E[ii].dof()
+        if atonce:
+            for ii in range(self.num_states):
+                degrees_of_freedom += 4.5
```

Comment:
Add a comment here that this assumes you're using the cell cycle model, because otherwise the tHMM code works for any type of emissions.

Reply:
done.